### PR TITLE
fix: github ci variables

### PR DESCRIPTION
**Pull request by carlocorradini**
Last commit to fix GitHub variables
Thanks again 🥳🤯🙏

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -40,8 +40,10 @@ jobs:
           fetch-depth: 0
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v2
-      - name: Extract VERSION from tag
-        run: echo "VERSION=${GITHUB_REF#refs/tags/v}" >> $GITHUB_ENV
+      - name: Extract VERSION
+        run: |
+          _version=$(printf '%s\n' "${{ github.ref_name }}" | sed 's/^.\{1\}//')
+          printf 'VERSION=%s\n' "$_version" >> "$GITHUB_ENV"
```
Comment (felipecrs) [inline .github/workflows/ci.yaml:46]: isn't the old version simpler?

**Comment by carlocorradini** [comment]
Is is better to use `github.ref_name` 🤯

**Comment by felipecrs** [comment]
> Is is better to use `github.ref_name` 🤯

Why?

**Comment by carlocorradini** [comment]
> > Is is better to use `github.ref_name` 🤯
> 
> Why?

Why not?
It is to only be consistent with the other workflow (`new.yaml`)

What do you prefer ahha

**Comment by felipecrs** [comment]
No longer needed after #9. I plan to tackle automatic updates in a different way btw.

**Pull request closed without merging**